 sdk/wxconfig/wx-config-win.cpp | 10 ++++++++--
 * Make wx-config to honor --libs <comma-list> in the middle of the command line argument

diff --git a/sdk/wxconfig/wx-config-win.cpp b/sdk/wxconfig/wx-config-win.cpp
index 3839cea..48db72b 100644
--- a/sdk/wxconfig/wx-config-win.cpp
+++ b/sdk/wxconfig/wx-config-win.cpp
@@ -326,9 +326,15 @@ public:
             std::string line;
             line = argv[i];
             split(line);
+			
+			if( line == "--libs" ) {
+				std::string libsList;
+				if( (i + 1) < argc ) {
+					libsList = argv[i+1];
+				}
+				parseLibs(libsList);
+			}
         }
-        parseLibs(argv[argc-1]);
-
         return true;
     }
 
